docs/getting_started_guide/register.rst

```diff
@@ -111,24 +111,40 @@ to each provider supported by ``eodag``:
 
   * Add your WEkEO credentials (*username*, *password*) to the user configuration file.
 
-  * You will then need to accept Copernicus terms and conditions (for once). To do this, follow the
+  * Depending on which data you want to retrieve, you will then need to accept terms and conditions (for once). To do this, follow the
     `tutorial guidelines <https://eodag.readthedocs.io/en/latest/notebooks/tutos/tuto_wekeo.html#Registration>`__
     or run the following commands in your terminal.
 
   * First, get a token from your base64-encoded credentials (replace USERNAME and PASSWORD with your credentials):
 
     .. code-block:: bash
 
-      curl -X GET --header "Authorization: Basic $(echo USERNAME:PASSWORD | base64)" "https://wekeo-broker.prod.wekeo2.eu/databroker/gettoken"
+      curl -X POST --data '{"username": "juser", "password": "password"}' -H "Content-Type: application/json" "https://gateway.prod.wekeo2.eu/hda-broker/gettoken"
 
     The WEkEO API will respond with a token:
 
     .. code-block:: bash
 
-      { "access_token": "xxxxxxxx-yyyy-zzzz-xxxx-yyyyyyyyyyyy" }
+      { "access_token": "xxxxxxxx-yyyy-zzzz-xxxx-yyyyyyyyyyyy",
+        "refresh_token": "xxxxxxxx-yyyy-zzzz-xxxx-yyyyyyyyyyyy",
+        "scope":"openid",
+        "id_token":"token",
+        "token_type":"Bearer",
+        "expires_in":3600
+      }
 
-  * Accept terms and conditions by running this command and replacing <access_token>:
+  * Accept terms and conditions by running this command and replacing <access_token> and <licence_name>:
 
     .. code-block:: bash
 
-      curl --request PUT --header 'accept: application/json' --header 'authorization: <access_token>' --data 'accepted=true' https://wekeo-broker.prod.wekeo2.eu/databroker/termsaccepted/Copernicus_General_License
+      curl --request PUT --header 'accept: application/json' --header 'Authorization: Bearer <access_token>' https://gateway.prod.wekeo2.eu/hda-broker/api/v1/termsaccepted/<licence_name>
+
+   The licence name depends on which data you want to retrieve. To use all datasets available in wekeo, the following licences have to be accepted:
+
+    * EUMETSAT_Copernicus_Data_Licence
+    * Copernicus_Land_Monitoring_Service_Data_Policy
+    * Copernicus_Sentinel_License
+    * Copernicus_ECMWF_License
+    * Copernicus_Marine_Service_Product_License
+    * Copernicus_DEM_Instance_COP-DEM-GLO-30-F_Global_30m
+    * Copernicus_DEM_Instance_COP-DEM-GLO-90-F_Global_90m
```

Comment:
please fix indentation

Reply:
indentation should be updated to fix the display in https://eodag--1034.org.readthedocs.build/en/1034/getting_started_guide/register.html

Reply:
fixed